assets/uswds/js/uswds.js

```diff
@@ -5070,7 +5070,7 @@ object-assign
         const NAV_LINKS = `${NAV} a`;
         const NON_NAV_HIDDEN_ATTRIBUTE = `data-nav-hidden`;
         const OPENERS = `.${PREFIX}-menu-btn`;
-        const CLOSE_BUTTON = `.${PREFIX}-nav-close`;
+        const CLOSE_BUTTON = `.${PREFIX}-nav__close`;
```

Comment:
This is still unchanged. Can you revert this to its original state?

Reply:
wdym? This is the original uswds.js... isn't it?

Reply:
I'm no longer seeing assets/uswds/js/uswds.js being in the changed files at all. Weird. Must have been a cache issue or something. Disregard.